packages/@coorpacademy-components/src/atom/gradient/index.native.tsx

```diff
@@ -24,7 +24,7 @@ const Gradient = ({
 }: Props) => {
   let calculatedColors = colors;
   if (colors.length === 1) {
-    const {r, g, b} = Color(colors[0]).object();
+    const {r, g, b} = new Color(colors[0]);
     calculatedColors =
       transparencyPosition === 'top'
         ? [`rgba(${r}, ${g}, ${b}, 0)`, colors[0], colors[0]]
```

Comment:
à la place de `rgba(${r}, ${g}, ${b}, 0)`, tu peux utiliser `.toString({format: "rgba"});`
:link: https://colorjs.io/docs/output#getting-a-string-representation-of-a-color